lib/Text/Glob.pm

```diff
@@ -177,6 +189,10 @@ the regex, or change the seperator that Text::Glob uses by setting
 C<$Text::Glob::seperator> to an alternative value while compiling the
 the regex.
 
+However, if C<$Text::Glob::globstar> is set to a true value, two
+consecutive asterisks (e.g. C<**.foo>) B<will> match C</>, though one
+asterisk will not.
+
```

Comment:
For discussion, see my proposal at https://github.com/nrdvana/perl-text-glob/commit/7f7bfb983a4dd74524d3fde4c0b741189461e7a6

The text says "bash behavior" but my examples are actually showing rsync behavior.  In order to become bach compatible, ``**.foo`` would need to instead be ``**/*.foo`` and would somewhat surprisingly be able to match the string ``bar.foo``.